Will run all tests of outer class when trigger inner class in test explorer

Normal when triggering the test of inner class in a test file
But when trigger inner class in test explorer, it will run all tests including other methods under the outer class
![image](https://user-images.githubusercontent.com/22470312/48122543-be538200-e2b2-11e8-9de0-128ce969be1a.png)

There is another issue here:

I tested the inner class case in a TestNG project, the TestNG runner cannot find the classpath, which says:
```
[ERROR] org.testng.TestNGException: 
Cannot find class in classpath: com.mkyong.testng.examples.suite.TestDatabase.InnerClass
	at org.testng.xml.XmlClass.loadClass(XmlClass.java:81)
	at org.testng.xml.XmlClass.init(XmlClass.java:73)
	at org.testng.xml.XmlClass.init(XmlClass.java:63)
	at org.testng.xml.XmlClass.<init>(XmlClass.java:39)
	at com.microsoft.java.test.runner.testng.TestNGRunner.createClass(TestNGRunner.java:52)
	at com.microsoft.java.test.runner.testng.TestNGRunner.createTests(TestNGRunner.java:46)
	at com.microsoft.java.test.runner.testng.TestNGRunner.run(TestNGRunner.java:31)
	at com.microsoft.java.test.runner.testng.TestNGLauncher.execute(TestNGLauncher.java:32)
	at com.microsoft.java.test.runner.Launcher.main(Launcher.java:53).
```

--- a/java-extension/com.microsoft.java.test.plugin/src/main/java/com/microsoft/java/test/plugin/util/TestSearchUtils.java
+++ b/java-extension/com.microsoft.java.test.plugin/src/main/java/com/microsoft/java/test/plugin/util/TestSearchUtils.java
@@ -181,9 +181,9 @@ public class TestSearchUtils {
                 final Object element = match.getElement();
                 if (element instanceof IMethod) {
                     final IMethod method = (IMethod) element;
-                    // Fix bug https://github.com/Microsoft/vscode-java-test/issues/441
-                    // - The SearchEngine doesn't honer method level search scope.
-                    if (params.getLevel() == TestLevel.METHOD && !scope.encloses(method)) {
+                    // The search result might not in the search scope.
+                    // See: https://github.com/Microsoft/vscode-java-test/issues/441
+                    if (!scope.encloses(method)) {
                         return;
                     }
                     final TestItem methodItem = TestFrameworkUtils.resoveTestItemForMethod(method);

